test/compiler.test.js

```diff
@@ -613,6 +613,31 @@ describe('compiler', function() {
       });
     });
 
+    it('resolves non-relative path in `modelSources` option', function() {
+      appdir.createConfigFilesSync({}, {}, {
+        Car: { dataSource: 'db' }
+      });
+      appdir.writeConfigFileSync('custom-models/car.json', { name: 'Car' });
+      appdir.writeFileSync('custom-models/car.js', '');
+
+      var instructions = boot.compile({
+        appRootDir: appdir.PATH,
+        modelSources: ['custom-models']
+      });
+
+      expect(instructions.models).to.have.length(1);
+      expect(instructions.models[0]).to.eql({
+        name: 'Car',
+        config: {
+          dataSource: 'db'
+        },
+        definition: {
+          name: 'Car'
+        },
+        sourceFile: path.resolve(appdir.PATH, 'custom-models', 'car.js')
+      });
```

Comment:
Same comment here, since we care about the resolved path only, can you please simplify the second check to the following?

```js
expect(instructions.models[0].sourceFile).to.equal(appdir.resolve('custom-models', 'car.js'));
```